pkgs/by-name/hy/hyprnotify/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  pkgs,
+  fetchFromGitHub,
+}:
+
+pkgs.buildGoModule rec {
+  pname = "hyprnotify";
+  version = "0.7.0";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "codelif";
+    repo = "hyprnotify";
+    rev = "v${version}";
+    hash = "sha256-dL+W+iMwRNw9042bs2XUFPMCCqIvDENXOMzhcLh+RL4=";
+  };
+
+  nativeBuildInputs = with pkgs; [ pkg-config ];
+  buildInputs = with pkgs; [ alsa-lib ];
+  vendorHash = "sha256-AZDtaiSNq7em876Q9f+YeDxboqVwA8IE9dDM6zggFXs=";
+
+  meta = with lib; {
```

Comment:
`with lib` is not such a good idea. We're already trying to reduce it as much as possible.